### ISSUE TITLE
Numbers in url would be taken as bug index and doing wrong replacement on reporting.

### ISSUE THREAD

**Issue by astleychen** [astleychen/status-bot#3]
For example:
**irc event:** 

`standups: submitted https://github.com/servo/servo/pull/16806`

will be processed to:
```
* {{bug|16806}} Invalid bug or security bug
** Submitted https://github.com/servo/servo/pull/bug 16806
```

**Comment by astleychen** [comment]
Fixed in commit: 1eb739757577184615a8940e8aec051ccce9e673

By current design, a number of digits, say d{5,} would only be recognized as bugzilla number follows the syntax as below: "[Bb]ug \\d+"

**Comment by astleychen** [comment]
Closed this bug.
